packages/analytics-js-plugins/src/errorReporting/utils.ts

```diff
@@ -17,6 +17,7 @@ import type {
 import { generateUUID } from '@rudderstack/analytics-js-common/utilities/uuId';
 import { METRICS_PAYLOAD_VERSION } from '@rudderstack/analytics-js-common/constants/metrics';
 import { stringifyWithoutCircular } from '@rudderstack/analytics-js-common/utilities/json';
+import { ERROR_MESSAGES_TO_BE_FILTERED } from '@rudderstack/analytics-js-common/constants/errors';
```

Comment:
It doesn't make sense to be part of the common package anymore.

Reply:
Please resolve it